src/modify.js

```diff
@@ -66,12 +75,31 @@ function rewriteAllFields(schema, configCallback, context) {
   });
 }
 
+function reorderFields(schema, orderCallback) {
+  if (!orderCallback) return null;
+
+  const originalOrder = schema['x-jsf-order'];
+  const orderConfig = orderCallback(originalOrder);
+  const remaining = difference(originalOrder, orderConfig.order);
+
+  const finalOrder =
+    orderConfig.rest === 'start'
```

Comment:
It took me some time to understand what `rest` means. Have you thought about using other parameter name? Suggestion: `remainingFieldsPosition`?

Reply:
Can we find a middle name that is not so long, but not so short? What about `leftovers` or `remaining`

Reply:
Tbh I would rather use a more descriptive name. Putting myself in a developer's shoes, none of these field names seem intuitive. That's why I suggest a longer name.

Reply:
We had a sync chat, and we concluded to remove `rest` attr all together. If the devs wants to customize this they can always use the callback `(originalOrder) => {}` to apply any desired manipulation